### ISSUE TITLE
About pretrain the YOLOV++ models

### ISSUE THREAD

**Issue by KaihongLi** [YuHengsss/YOLOV#97]
Please ask is to pre-train YOLOX by ImageNet VID and DET at the same time, and then fine-tune the model by ImageNet VID to get the final

**Comment by YuHengsss** [comment]
Yep, the pipeline of YOLOV++ is the same as YOLOV. The differences lie in the model arc.
